Configuration/Eras/python/Era_Run2_2016_cff.py

```diff
@@ -5,6 +5,9 @@
 from Configuration.Eras.Modifier_stage2L1Trigger_cff import stage2L1Trigger
 from Configuration.Eras.Modifier_ctpps_2016_cff import ctpps_2016
 from Configuration.Eras.Modifier_run2_HLTconditions_2016_cff import run2_HLTconditions_2016
+from Configuration.Eras.Modifier_run2_muon_2016_cff import run2_muon_2016
 
-Run2_2016 = cms.ModifierChain(run2_common, run2_25ns_specific, stage2L1Trigger, ctpps_2016, run2_HLTconditions_2016)
+
+
+Run2_2016 = cms.ModifierChain(run2_common, run2_25ns_specific, stage2L1Trigger, ctpps_2016, run2_HLTconditions_2016, run2_muon_2016)
```

Comment:
here and in other files please add a linebreak for readability and drop the extra newline(s) before "Run2_2016 = "